src/client/cockpit-client

```diff
@@ -295,24 +294,17 @@ class ExternalCockpitWs:
 
 class InternalCockpitWs:
     def start(self):
-        self.socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-        # Static port is needed for WebKit local storage to persist across runs
-        self.socket.bind(('127.0.90.90', 9090))
-        self.socket.listen()
-
         self.write_config()
-        self.ws = subprocess.Popen([f'{libexecdir}/cockpit-ws'],
+        host = '127.0.0.90'
+        port = '9090'
```

Comment:
Could have been an int...

What if you made a typo and accidentally typed in something that wasn't a number?!  😅

Reply:
I checked it very carefully and proved by incomplete induction that 9090 is, in fact, a natural number! :nerd_face: